scripts/examples_master.py

```diff
@@ -615,11 +615,16 @@
                 },
                 # Text generation
                 {
-                    "path": "text_generation_gpt",
-                    "title": "GPT text generation with KerasNLP",
+                    "path": "gpt2_text_generation_with_kerasnlp",
```

Comment:
One weird thing to deal with, our guide is currently sitting under `nlp` not `generative`. That means with this change it will list twice! Should we just move it under generative?

Reply:
Moved for now, seems most consistent.